### DISCUSSION TITLE
Will you port Quickemu to run on WIndows natively?

### DISCUSSION THREAD

**Discussion by TechElliott** [quickemu-project/quickemu#894] [Quickemu] [answered]
I was a Linux user until I needed a program only on Windows. I love this program and I prefer this program over VMWare or VirtualBox. I would love for a windows version to be available. And yes, I tried this on WSL and it did not work. If it is even possible, please port this to Windows

**Comment by lj3954** [comment]
This likely is better suited for the issues tab, as it's a feature request. It's not possible though, or at least, it's out of scope for this project. Quickemu uses QEMU, which is accelerated by KVM or other kernel-level virtualization. Windows doesn't support anything like this, and software virtualization is far too slow to be of any use.

**Reply by shawny43** [reply]
Qemu is already available has installation on Windows, what's does you mean for it?

**Reply by lj3954** [reply]
QEMU is incredibly slow without KVM, since it must fall back to software virtualization. In addition, the entire quickemu project is a collection of bash scripts. It would require a complete rewrite to be compatible with Windows in any way.

**Reply by shawny43** [reply]
Has i know there WSL2 that may allow KVM inside but i think it's will acting like an HyperV bridging an KVM inside WSL.

**Reply by lj3954** [reply]
WSL is just a virtual machine running GNU/Linux. If nested virtualization doesn't work through that, you need to go through a support channel for Windows operating systems in order to find a way to enable it. Quickemu is unrelated to this issue.

**Comment by flexiondotorg** [comment, marked as answer]
I have no plans to adapt Quickemu to run natively on Windows. We are interested in improving the support for running Quickemu on WSL2 as the host.

**Reply by shawny43** [reply]
Glad we found an alternative solution. Hope this gonna work.
